espnet2/tasks/svs.py

```diff
@@ -288,8 +288,7 @@ def build_preprocess_fn(
             )
         else:
             retval = None
-        # FIXME (jiatong): sometimes checking is not working here
-        # assert check_return_type(retval)
+        print(type(retval))
```

Comment:
@ftshijt, is it OK?

Reply:
Sorry, I think that this print was a mistake. I maybe checking the type of the variable. I will remove the line